tests/models/test_structures.py

```diff
@@ -38,4 +38,45 @@ def test_bad_structures(bad_structures, mapper):
         # This is for helping devs finding any errors that may occur
         print(f"Trying structure number {index} from 'test_bad_structures.json'")
         with pytest.raises(ValidationError):
-            StructureResource(**mapper(MAPPER).map_back(structure))
+            try:
+                StructureResource(**mapper(MAPPER).map_back(structure))
+            except Exception as exc:
+                print(exc)
+                raise exc
```

Comment:
Is this still necessary?
```suggestion
            StructureResource(**mapper(MAPPER).map_back(structure))
```

Reply:
This was just another convenience (as with the print a few lines above) to keep the structure number and its error on the same screen in stdout (rather than pytest separating them). I don't think this does any harm

Reply:
But it should already print out the exception if any occurs, right?

Reply:
It does but: 1) pytest clips the long validation error messages unless you run with `-vvvv`, 2) pytest separates stderr and stdout so you have to scroll a fair way back to get the structure index output

Reply:
See e.g. https://github.com/Materials-Consortia/optimade-python-tools/blob/b96c462568dd8c09047fbfb266a08fe7922809e6/tests/models/test_structures.py#L24-L32

I won't do the git blame to see which one of us added this back in the day :P

Reply:
Ah damnit, I've been reading the test_good_structures function and not the one you're referring to

Reply:
My bad, you're completely right, this is useless unless it raises something other than a validation error

Reply:
Just to answer this, although I see you've already changed this.

> It does but: 1) pytest clips the long validation error messages unless you run with `-vvvv`, 2) pytest separates stderr and stdout so you have to scroll a fair way back to get the structure index output, at which points its nice to see the error too

This is true, but seems more of an issue of how `pytest` presents its results by default than how we should write our tests?
Hence, we should try and present any results in a better way by adding an extra `pytest` summary section or similar?